proofs/cbmc/crypto_sign_verify_internal/Makefile

```diff
@@ -0,0 +1,66 @@
+# Copyright (c) The mldsa-native project authors
+# SPDX-License-Identifier: Apache-2.0 OR ISC OR MIT
+
+include ../Makefile_params.common
+
+HARNESS_ENTRY = harness
+HARNESS_FILE = crypto_sign_verify_internal_harness
+
+# This should be a unique identifier for this proof, and will appear on the
+# Litani dashboard. It can be human-readable and contain spaces if you wish.
+PROOF_UID = crypto_sign_verify_internal
+
+DEFINES +=
+INCLUDES +=
+
+REMOVE_FUNCTION_BODY +=
+UNWINDSET +=
+
+PROOF_SOURCES += $(PROOFDIR)/$(HARNESS_FILE).c
+PROJECT_SOURCES += $(SRCDIR)/mldsa/sign.c
+
+CHECK_FUNCTION_CONTRACTS=$(MLD_NAMESPACE)crypto_sign_verify_internal
+USE_FUNCTION_CONTRACTS=$(MLD_NAMESPACE)unpack_pk $(MLD_NAMESPACE)unpack_sig $(MLD_NAMESPACE)polyvecl_chknorm\
+                       $(FIPS202_NAMESPACE)shake256 $(FIPS202_NAMESPACE)shake256_init \
+                       $(FIPS202_NAMESPACE)shake256_absorb $(FIPS202_NAMESPACE)shake256_finalize \
+					   $(FIPS202_NAMESPACE)shake256_squeeze \
+					   $(MLD_NAMESPACE)poly_challenge $(MLD_NAMESPACE)polyvec_matrix_expand\
+					   $(MLD_NAMESPACE)polyvecl_ntt $(MLD_NAMESPACE)polyvec_matrix_pointwise_montgomery \
+					   $(MLD_NAMESPACE)poly_ntt $(MLD_NAMESPACE)polyveck_shiftl \
+					   $(MLD_NAMESPACE)polyveck_ntt $(MLD_NAMESPACE)polyveck_pointwise_poly_montgomery \
+                       $(MLD_NAMESPACE)polyveck_sub $(MLD_NAMESPACE)polyveck_reduce \
+                       $(MLD_NAMESPACE)polyveck_invntt_tomont $(MLD_NAMESPACE)polyveck_caddq \
+                       $(MLD_NAMESPACE)polyveck_use_hint $(MLD_NAMESPACE)polyveck_pack_w1
+	   
+APPLY_LOOP_CONTRACTS=on
+USE_DYNAMIC_FRAMES=1
+
+# Disable any setting of EXTERNAL_SAT_SOLVER, and choose SMT backend instead
+EXTERNAL_SAT_SOLVER=
+CBMCFLAGS=--smt2
```

Comment:
Suggest you add --slice-formula here. It usually helps.

Reply:
done